### DISCUSSION TITLE
Schematic question

### DISCUSSION THREAD

**Discussion by ej012345** [picocomputer/community#6] [Q&A] [answered]
Please help me understand the schematic, does this show pin 8 of 74HC30  to pin 5 of 74AC02?
![image](https://github.com/picocomputer/discussions/assets/131208326/510e9256-92e1-41fc-9a5e-a615fe74c698)
In the below picture it looks like Pin 8 to Pin 8?
![image](https://github.com/picocomputer/discussions/assets/131208326/5bb0f4f8-5891-4d7e-973e-d273603e9434)
I must admit, the Jx connections do confuse me.
Thanks, EJ

**Comment by rumbledethumps** [comment, marked as answer]
It doesn't matter which gate you use.

**Reply by ej012345** [reply]
Got it, U7C is the same as U7B. Follow the schematic and I’ll be good!
